From 1504dcfd7df262b5c3634c1923ed83efdd54dd57 Mon Sep 17 00:00:00 2001
From: mengbinghan <binghan.meng@spacemit.com>
Date: Tue, 5 Nov 2024 10:08:45 +0800
Subject: [PATCH 1067/1206] vpu: Fix amvx build error when building amvx as
 module

log:
ERROR: modpost: module amvx uses symbol dma_buf_unmap_attachment_unlocked from namespace DMA_BUF, but does not import it.
ERROR: modpost: module amvx uses symbol dma_buf_attach from namespace DMA_BUF, but does not import it.
ERROR: modpost: module amvx uses symbol dma_buf_map_attachment_unlocked from namespace DMA_BUF, but does not import it.
ERROR: modpost: module amvx uses symbol dma_buf_get from namespace DMA_BUF, but does not import it.
ERROR: modpost: module amvx uses symbol dma_buf_put from namespace DMA_BUF, but does not import it.
ERROR: modpost: module amvx uses symbol dma_buf_detach from namespace DMA_BUF, but does not import it.
make[4]: *** [scripts/Makefile.modpost:145: Module.symvers] Error 1
make[3]: *** [Makefile:1873: modpost] Error 2

Change-Id: I63f4a1127deb36e3ba66e20820ed61f7835fdf9a
---
 drivers/media/platform/spacemit/vpu_k1x/if/mvx_mmu.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/media/platform/spacemit/vpu_k1x/if/mvx_mmu.c b/drivers/media/platform/spacemit/vpu_k1x/if/mvx_mmu.c
index a6d174a03b4f..28c08fe617f9 100644
--- a/drivers/media/platform/spacemit/vpu_k1x/if/mvx_mmu.c
+++ b/drivers/media/platform/spacemit/vpu_k1x/if/mvx_mmu.c
@@ -44,6 +44,8 @@
 #include "mvx_mmu.h"
 #include "mvx_log_group.h"
 
+MODULE_IMPORT_NS(DMA_BUF);
+
 /****************************************************************************
  * Defines
  ****************************************************************************/
-- 
2.47.0

